packages/editor-ui/src/store.ts

```diff
@@ -597,6 +599,11 @@ export const store = new Vuex.Store({
 			Vue.set(state, 'nodeTypes', updatedNodes);
 			state.nodeTypes = updatedNodes;
 		},
+
+		addSidebarMenuItems (state, menuItems: IMenuItem[]) {
+			const updated = [...state.sidebarMenuItems, ...menuItems];
```

Comment:
I think this would be more readable + is more performant
```suggestion
			const updated = state.sidebarMenuItems.concat(menuItems);
```